frontend/src/metabase/selectors/embed.ts

```diff
@@ -1,7 +1,9 @@
+import { createSelector } from "@reduxjs/toolkit";
+
 import { isWithinIframe } from "metabase/lib/dom";
 import type { InteractiveEmbeddingOptions, State } from "metabase-types/store";
 
-export const getIsEmbedded = (_state?: State): boolean => {
+export const getIsEmbeddingIframe = (_state?: State): boolean => {
```

Comment:
make the name resembling `getIsEmbeddingSdk` following the structure `getIsEmbedding<SubType>`